src/main/java/com/labsynch/labseer/service/BulkLoadServiceImpl.java

```diff
@@ -894,6 +897,9 @@ public Parent validateParent(Parent parent, String chemist, Collection<BulkLoadP
 							// No Changes or Modifications Needed; Can Break Loop
 							// Continue 
 							parent = foundParent;
+							// Add the multiple fragments boolean back to the parent
+							// from the transient property we calculated earlier
+							parent.setMultipleFragments(multipleFragments);
```

Comment:
This part makes sense to me - if it's the same structure, we can copy in this property without hitting BBChem again.